.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -84,10 +84,10 @@ To do this you will need a synchronized `zcashd` node. You can request help from
 **Important**: Any merge into `main` deletes any edits to the draft changelog.
 Once you are ready to tag a release, copy the draft changelog into `CHANGELOG.md`.
 
-We follow the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) format.
```

Comment:
I need to look at the new release drafter workflow and changelogs but do we not use the Keep a Changelog format anymore?